tests/unit/model_airdata_test.cpp

```diff
@@ -3,21 +3,30 @@
 
 extern "C" {
 #include "application/estimator/model/model_airdata.h"
-}
-
 // 0.005% tolerance, proportional to the actual value
 #define tolerance 0.00005
+#define TOLERANCE 0.000001 // non-ratio, checks 6 decimals points
```

Comment:
> #define tolerance
> #define TOLERANCE


cmon...

Reply:
The fact that you went through the effort to write verbose comments and not to rename them is... sure something